samples/algorithms/BitFlipCode.qs

```diff
@@ -119,14 +119,14 @@ namespace Sample {
         // parity measurements.
         let indexOfError =
             if (parity01, parity12) == (One, Zero) {
```

Comment:
I think if we just change our sample so that `let indexOfError = if (parity01, parity12) .....` is on one line instead of being broken up with a newline, it works better with the current behavior.

Reply:
Formatting of the library and samples has been removed from this PR. They may be added in a near future PR.